client/src/pages/Signup/index.js

```diff
@@ -0,0 +1,3 @@
+export * from "./SignupPage";
+export * from "./SignupSuccessPage";
+export * from "./SignupWarningPage";
```

Comment:
I don't think the code for this page:
<img width="536" alt="image" src="https://user-images.githubusercontent.com/84544729/204343831-b134a6bc-8a7b-4e61-8a56-50379374dde9.png">

Also the text is a bit wrong ("the beta access for the GreenLake for Data Fabric"). Maybe just "Sorry, we can't activate the GreenLake for Data Fabric beta access".